book/src/tutorial/signing.md

```diff
@@ -0,0 +1,114 @@
+# Signing
+
+The diagram below shows the signing process. Dashed lines represent data being
+sent through an [authenticated communication
+channel](https://frost.zfnd.org/terminology.html#peer-to-peer-channel).
+
+![Diagram of Signing, illustrating what is explained in the text](signing.png)
+
+## Coordinator, Round 1
+
+To sign, the
+[Coordinator](file:///home/conrado/zfnd/frost/book/book/frost.html#signing) must
+select which participants are going to generate the signature, and must signal
+to start the process. This needs to be implemented by users of the ZF FROST library and will depend on
+the communication channel being used.
+
+## Participants, Round 1
+
+Each selected participant will then generate the nonces (a `SigningNonces`) and
+their commitments (a `SigningCommitments`) by calling
+[`round1::commit()`](https://docs.rs/frost-ristretto255/latest/frost_ristretto255/round1/fn.commit.html):
+
+```rust,no_run,noplayground
+{{#include ../../../frost-ristretto255/README.md:round1_commit}}
+```
+
+The `SigningNonces` must be kept by the participant to use in Round 2, while the
+`SigningCommitments` must be sent to the Coordinator using an [authenticated
+channel](https://frost.zfnd.org/terminology.html#broadcast-channel).
+
```

Comment:
When sending the `round1::SigningCommitments` to the Coordinator, this 'just' needs to be an authenticated peer-to-peer channel, not a broadcast channel, right? 

```suggestion
The `SigningNonces` must be kept by the participant to use in Round 2, while the
`SigningCommitments` must be sent to the Coordinator using an [authenticated
channel](https://frost.zfnd.org/terminology.html#peer-to-peer-channel).

```

Reply:
Right, wrong link, thanks!